### PR TITLE
Translation to Spanish

### PR DESCRIPTION
**Pull request by h771864**
Hi! I hope to do better this time :-)

### CODE REVIEW

File: Mods/DefensivePositions/Languages/Spanish/DefInjected/KeyBindings.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<KeyBindingDefs>
```
Comment (UnlimitedHugs) [inline Mods/DefensivePositions/Languages/Spanish/DefInjected/KeyBindings.xml]: Should be <LanguageData>

File: Mods/DefensivePositions/Languages/Spanish/DefInjected/KeyBindingCategories.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<KeyBindingCategoryDefs>
```
Comment (UnlimitedHugs) [inline Mods/DefensivePositions/Languages/Spanish/DefInjected/KeyBindingCategories.xml]: For DefInjected text we don't copy over the whole def, we just specify the text fields to be replaced like so:
`<defName.fieldName>translated text</defName.fieldName>`
Which means, this file should contain instead:
```
<LanguageData>
	<DefensivePositions.description>Accesos directos para las acciones del mod Defensive Positions.</DefensivePositions.description>
</LanguageData>
```

File: Mods/DefensivePositions/Languages/Spanish/DefInjected/KeyBindings.xml (hunk 1 above)
Comment (UnlimitedHugs) [inline Mods/DefensivePositions/Languages/Spanish/DefInjected/KeyBindings.xml]: See comment on KeyBindingCategories.xml, same issue

**Comment by h771864** [comment]
Now works well, I regret the inconvenience and thanks for your patience.

**Pull request merged**